tests/IBFE/interpolate_velocity_01.c.output

```diff
@@ -0,0 +1,12 @@
+Number of elements: 10870
+
+IBFEMethod: mesh part 0 is using FIRST order LAGRANGE finite elements.
+
+IBHierarchyIntegrator::initializePatchHierarchy(): tag_buffer = 1 3
+FEDataManager::buildGhostedSolutionVector(): building ghosted solution vector for system: IB coordinates system
+FEDataManager::buildGhostedSolutionVector(): constructing ghost DOF index list for system: IB coordinates system
+FEDataManager::buildGhostedSolutionVector(): building ghosted solution vector for system: IB force system
+FEDataManager::buildGhostedSolutionVector(): constructing ghost DOF index list for system: IB force system
+FEDataManager::buildL2ProjectionSolver(): building L2 projection solver for system: IB velocity system
+max vertex distance: 0.00892356
+max norm errors: 2.754841494412296754e-05   3.9301631085009169908e-05
```

Comment:
These max norm errors are correct up to ten digits which (while we could improve this) is much more reasonable.